File: PersonnelManagement/PersonnelManagement/Models/Persondecree.cs
using System;
using System.Collections.Generic;

namespace PersonnelManagement.Models
{
    public partial class Persondecree
    {
        public int Id { get; set; }
        public DateTime Datecreated { get; set; }
        public DateTime? Datesigned { get; set; }
        public int Creator { get; set; }
        public int Owner { get; set; }
        public string Name { get; set; }
        public string Nickname { get; set; }
        public string Number { get; set; }
        public string Numbertype { get; set; }
        public sbyte Transfer { get; set; }
        public sbyte Signed { get; set; }
        public int Persondecreelevel { get; set; }
        public int Mailexplorerid { get; set; }
    }
}

File: PersonnelManagement/PersonnelManagement/Front/classes/persondecree.ts
import User from './user';

export default class Persondecree {
    persondecreeManagementStatus: number; // 1 - Создать новый проект приказа, 2 - отменить проект приказа, 3 - подписать проект приказа, 4 - распечатать приказ
                                          // 5 - обновить информацию проекта приказа (номер, дата), 6 - предоставить к общему доступу/закрыть общий доступ.
    id: number;
    datecreated: Date;
    datesigned: Date;
    creator: number;
    owner: number;
    name: string;
    nickname: string;
    number: string;
    numbertype: string;
    transfer: number;
    persondecreelevel: number;
    signed: number;
    mailexplorerid: number;

    creatorObject: User;

    creatorfolder: number;
    ownerfolder: number;
    accessforreading: string;

    // Только на стороне фронтэнда
    marked: boolean;

    getFIO: string;
    getPlace: string;
    getDate: string;
    getName: string;
    getNumber: string;
}